### ISSUE TITLE
Empty white area on Firefox when displayed in overflow menu

### ISSUE THREAD

**Issue by SuNNjek** [passbolt/passbolt_browser_extension#92]
## Empty white area on Firefox when displayed in overflow menu
* Passbolt Version: 2.12.0
* Platform and Target:
-- Browser: Firefox 73.0.1
-- Web server: Self-hosted 2.12.0

### What you did
I moved the extension icon to the overflow menu because I don't use it that often and I don't want my browser toolbar to be cluttered.

### What happened
When I open the extension from the overflow menu, the extension gets displayed in the size of the overflow menu. Because of this the menu is displayed wider than it normally is. This excess space is not filled with any content and is just white, empty space.

Note the white area on the right in this screenshot:
![passbolt](https://user-images.githubusercontent.com/20151081/75675184-ed032200-5c86-11ea-9b42-b383e7ddf432.png)

### What you expected to happen
I would expect the menu to fill the full width available and fill the empty space.

**Comment by garrettboone** [comment]
When tested with the beastify Mozilla example extension (https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Your_second_WebExtension) the following was original:
![image](https://user-images.githubusercontent.com/47018474/85231573-41f9ae00-b3c6-11ea-96a7-42cd7ce8156d.png)
```
html, body {
  width: 100px;
}
```
With change:
![image](https://user-images.githubusercontent.com/47018474/85231548-f3e4aa80-b3c5-11ea-9131-ae700b31ea43.png)

```
html, body {
  width: calc(100%-1em);
}
```

I tried to take into account the scrollbar...though that may not be needed.

**Comment by cedricalfonsi** [comment]
Fixed with v3.3.0, thank you for your patience and support.
